ip-messaging/push-config/new-message-push-config/new-message-push-config.5.x.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+
+# Find your Account Sid at https://www.twilio.com/console/account/settings
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+# Create an API Key and Secret at https://www.twilio.com/console/chat/dev-tools/api-keys
+api_key = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+api_secret = 'your_api_secret'
+# Your Chat Service SID from https://www.twilio.com/console/chat/services
+service_sid = 'ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+# Authenticate with Twilio
+@client = Twilio::REST::Client.new(api_key, api_secret, account_sid)
```

Comment:
@hortega, I left a comment in another pull request, regarding the use of instance variables in the Ruby snippets. Would you mind taking a look at it, please? https://github.com/TwilioDevEd/api-snippets/pull/520#discussion_r141432264

I'd love to have your comments on this.

Reply:
Yes, I also agree the `@` is unnecesary.